pom.xml

```diff
@@ -119,7 +119,7 @@
             <dependency>
                 <groupId>com.github.cafdataprocessing</groupId>
                 <artifactId>util-email-content-segregation</artifactId>
-                <version>2.0.0-5</version>
+                <version>2.1.0-SCMOD-6825-SNAPSHOT</version>
```

Comment:
Don't forget to remove branch before merging.

Reply:
Removed branch build version